app/models/activity.rb

```diff
@@ -49,6 +51,14 @@ def tied_status
     UNTIED_TIED_STATUS_CODE
   end
 
+  def sector_category_name
```

Comment:
As you point out in the commit description, the right place for this presentiional logic is in the helper. I'm concerned that if the propoect of refactoring the activity controller is daunting now, it's going to make leak complexity elsewhere in the app with models and presenters sharing the responsibility of view logic. 

I would like to see if we can make this better and get it out of the model.

Reply:
There is a similar issue on the third party project PR - I wondered there is we use locales to pass the correct level strong - this might work here as well.

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/361#pullrequestreview-403360691